Info-Orbs/src/core/wifiWidget.cpp

```diff
@@ -15,36 +20,58 @@ void WifiWidget::setup() {
   m_manager.selectScreen(0);
   display.drawCentreString("Connecting" + m_connectionString, 120, 80, 1);
 
+  WiFi.mode(WIFI_STA); // For WiFiManager explicitly set mode to station, esp defaults to STA+AP
 
-  m_manager.selectScreen(1);
-  display.drawCentreString("Connecting to", 120, 80, 1);
-  display.drawCentreString("WiFi..", 120, 100, 1);
-  display.drawCentreString(WIFI_SSID, 120, 130, 1);
-
-  // Serial.println("Connecting to WiFi..");
-
-  WiFi.begin(WIFI_SSID, WIFI_PASS);
-
-  Serial.println("Connecting to WiFi..");
+  // reset settings - wipe stored credentials for testing
+  // these are stored by the esp WiFi library
+  // wifimgr.resetSettings();
 
+  // WiFiManager automatically connects using saved credentials,
+  // if connection fails, it starts an access point with the specified name ( "AutoConnectAP"),
+  // if empty will auto generate SSID, if password is blank it will be anonymous AP (wm.autoConnect())
+  // then goes into a blocking loop awaiting configuration and will return success result
 
+  // Set WiFiManager to non-blocking so status and info can be displayed
+  wifimgr.setConfigPortalBlocking(false);
+  // wm.setConfigPortalTimeout(60);
 
+  // wm.autoConnect(); // auto generated AP name from chipid
+  // wm.autoConnect("AutoConnectAP"); // anonymous AP
+  // wifimgr.autoConnect("AutoConnectAP","password"); // password protected AP
```

Comment:
There is quite some commented out code, leftover debug stuff?

Reply:
Yeah, taken from an example. It shows various styles. I used the simple option of an AP with no password. I left it in in case other users would prefer password protection, they'd know how to do it right in the code.